capstone-frontend/package.json

```diff
@@ -6,6 +6,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "deep-equal": "^2.0.3",
```

Comment:
Is there a reason why we need the "deep-equal" dependency? It seems that you don't use it on the React frontend

Reply:
I was testing out the package for use in checking equivalency between JSONs, and then forgot to remove it when I decided not to use it. Will remove.